src/main/java/seedu/address/logic/parser/event/EventCommandParser.java

```diff
@@ -45,6 +46,8 @@ public EventCommand parse(String args) throws ParseException {
         String eventOption = eventCommandMatcher.group("eventOption");
 
         switch (eventOption) {
+        case AddEventCommand.COMMAND_OPTION:
+            return new AddEventCommandParser().parse(args);
         default:
             throw new ParseException(EventCommand.OPTION_UNKNOWN);
```

Comment:
I think can update this part to display the correct flags as well so that it is consistent with ProfileCommandParser.
```java
throw new ParseException(EventCommand.OPTION_UNKNOWN + EventCommand.VALID_FLAGS);
```

Reply:
Agreed. 
```suggestion
            throw new ParseException(EventCommand.OPTION_UNKNOWN + EventCommand.VALID_FLAGS);
```